android/src/main/java/ee/forgr/capacitor_inappbrowser/InAppBrowserPlugin.java

```diff
@@ -321,6 +330,11 @@ public void openWebView(PluginCall call) {
       new WebViewCallbacks() {
         @Override
         public void urlChangeEvent(String url) {
+          String query = url.substring(url.indexOf("?") + 1);
```

Comment:
This is for a very specific usage, will not work with just any consumer of this plugin. We should leave a comment here for the use case